magenta/protobuf/music.proto

```diff
@@ -136,4 +145,22 @@ message NoteSequence {
     int32 instrument = 4;
     int32 program = 5;
   }
+
+  // Stores score-related information about a particular part (sometimes also
+  // called track or voice).
```

Comment:
do we have to include which staff this track/voice was part of to inject notes back into musescore?